src/components/CalendarHeader.tsx

```diff
@@ -46,6 +48,7 @@ function _CalendarHeader<T>({
       <View style={[u['z-10'], u['w-50'], borderColor]} />
       {dateRange.map((date) => {
         const _isToday = isToday(date)
+        const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : _isToday
         return (
```

Comment:
To reduce unnecessary function call, please do

```suggestion
      {dateRange.map((date) => {
        const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : isToday(date)
        return (
```